lib/cudalibmg/libcudalibmg.jl

```diff
@@ -0,0 +1,54 @@
+# Automatically-generated headers for cudalibmg
+#
+# DO NOT EDIT THIS FILE DIRECTLY.
+
+
+# Julia wrapper for header: cudalibmg.h
```

Comment:
wait, is there an actual cudalibmg? I thought this was a copy/paste error?

Reply:
There is in the CUBLASMG code ... maybe this has been hidden away somewhere in the DLL? Most of this code will only be called if you download and use CUBLASMG